bmq-sdk/src/test/java/com/bloomberg/bmq/impl/infr/util/expressionvalidator/TestExpressionValidator.java

```diff
@@ -0,0 +1,360 @@
+                    {"N1A2_M3E4 > 0", true, null},
+                    {"N1a2_m3e4_ > 0", true, null},
+                    {"n1a2m3e4_N5a6m7e8 > 0", true, null},
+                    {"N1a2m3e4_N5a6m7e8_ > 0", true, null},
+                    {"n0_ > 0", true, null},
+                    {"N_0 > 0", true, null},
+                    {"aB1_aB__2a___B3aB4____ > 0", true, null},
+                    {"aaa_111BBBaaa222__BBBaaa3_3_3BBB > 0", true, null},
+                    {"B_a_1_B_a_2_B_a_3_B_a_4_B_a_5_B_ > 0", true, null},
+
+                    // readable examples
+                    {"camelCase > 0", true, null},
+                    {"snake_case > 0", true, null},
+                    {"PascalCase > 0", true, null},
+                    {"price > 0", true, null},
+                    {"BetterLateThanNever > 0", true, null},
+                    {"firmId > 0", true, null},
+                    {"TheStandardAndPoor500 > 0", true, null},
+                    {"SPX_IND > 0", true, null},
+
+                    // all available characters
+                    {"abcdefghijklmnopqrstuvwxyz_0123456789 > 0", true, null},
+                    {"ABCDEFGHIJKLMNOPQRSTUVWXYZ_0123456789 > 0", true, null},
+
+                    // negative examples
+                    {"0name > 0", false, "syntax error, missed operation at offset 1"},
+                    {"1NAME > 0", false, "syntax error, missed operation at offset 1"},
+                    {"23456789Name > 0", false, "syntax error, missed operation at offset 8"},
+                    {
+                        "_nameName > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "0_NameName > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 1"
+                    },
+                    {
+                        "_1n > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "1_N > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 1"
+                    },
+                    {
+                        "_ > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {
+                        "_11111111111111111111111111111 > 0",
+                        false,
+                        "syntax error, unexpected invalid character \"_\" at offset 0"
+                    },
+                    {"22222222222222222222222222222_ > 0", false, "integer overflow at offset 0"},
+                });
+    }
+
+    @Test
+    public void expressionValidatorTest() {
```

Comment:
Usually, in this project, in test method name `test` is used as a prefix

Reply:
Updated as proposed